packages/styled-sheet/src/Sheet.js

```diff
@@ -0,0 +1,96 @@
+// @flow
+
+import type { GroupedTag, GroupedKeys } from './types';
+import { IS_BROWSER } from './constants';
+import { RuleGroupTag, DefaultTag, VirtualTag } from './tags';
+import { makeCssMarker, wrapInStyleTag } from './utils';
+import GroupRegistry from './GroupRegistry';
+import rehydrate from './rehydrate';
+
+// Ensure we only rehydrate once
+let SHOULD_REHYDRATE = !IS_BROWSER;
+
+const makeRuleGroupTag = (target?: HTMLElement, forceServer?: boolean): GroupedTag => {
+  const tag = forceServer ? new DefaultTag(target) : new VirtualTag();
```

Comment:
This condition is inverse, right? It should be

```suggestion
  const tag = !forceServer ? new DefaultTag(target) : new VirtualTag();
```